src/views/report/ScriptReportsOverview/index.tsx

```diff
@@ -134,6 +134,10 @@ const filterConfig: FilterConfig<Partial<IColumnNames>> = {
         label: <Translate msg="script_reports.overview.list.filter.run_id" />,
         filterType: FilterType.Search,
     },
+    runStatus: {
+        label: <Translate msg="script_reports.overview.list.filter.runStatus" />,
+        filterType: FilterType.Search,
```

Comment:
should be of type dropdown with all the options possible for run statusses. See environment above for inspiration